vars/newmanExecute.groovy

```diff
@@ -128,8 +164,33 @@ void call(Map parameters = [:]) {
                         config: config.plus([newmanCollection: collection]),
                         collectionDisplayName: collectionDisplayName
                     ]).toString()
+                def command_secrets
+                if(config.cfAppsWithSecrets){
+                    CfUtils cfUtils = new CfUtils();
+                    config.cfAppsWithSecrets.each { appName ->
+                        def xsuaaCredentials = cfUtils.getXsuaaCredentials(config.cloudFoundry.apiEndpoint,
+                                                    config.cloudFoundry.org,
+                                                    config.cloudFoundry.space,
+                                                    config.cloudFoundry.credentialsId,
+                                                    appName,
+                                                    config.verbose ? true : false )
```

Comment:
I think `config.verbose` would be sufficient.

Reply:
that won't work if `verbose` is undefined in yaml; because then the value will be `null` that cannot be taken as a `boolean` automatically.